src/utils/wsRedirect.ts

```diff
@@ -76,6 +78,7 @@ export class WsRedirect {
     logger.debug(`${messages.REDIRECT_CLOSING_WEB_SOCKET} to ${params.host}: ${params.port}.`)
     if (this.websocketFromDevice) {
       this.websocketFromDevice.CloseChannel()
+      MqttProvider.publishEvent('success', ['handleClose'], messages.REDIRECTION_SESSION_ENDED)
```

Comment:
Can this call fail? Is any error handling needed?

Reply:
If a MQTT broker is not configured/missing, the call will silently fail without impacting the rest of the code